.github/ISSUE_TEMPLATE/Bug_report.yml

```diff
@@ -1,5 +1,7 @@
 name: Bug Report
 description: Report an issue with the Postman platform.
+title: "[Bug] "
+labels: "bug"
```

Comment:
This change has a knock on effect with some automation that would create a new JIRA ticket each time someone raises a new issue. 

Not everything raised is a bug (We have a lot of random things raised using this template), we would need to triage the issue first before knowing that it's a bug.

Reply:
Interesting! Got it, so we use "bug" to basically mean confirmed bug and only add it once we've reproduced it or confirmed it.

Do you happen to know off-hand where that Jira automation is linked or documented?

Reply:
Removed this in [54dc97d](https://github.com/postmanlabs/postman-app-support/pull/12512/commits/54dc97da30d58e1938a5008c8ba5ca61a760a3d8)